dags/inference/maxtext_inference.py

```diff
@@ -35,14 +36,32 @@
 ) as dag:
   test_name_prefix = "maxtext-inference"
   test_models = {
-      "llama2-7b": "llama2_7b",
-      # "mistral": "test_mistral",
-      # "gemma-2b": "gemma/2b/test_gemma",
-      # "gpt3": "test_gpt3",
+      "llama2-7b": {
+          "per_device_batch_sizes": [1, 2, 4],
```

Comment:
FYI, the batch_sizes (and the ici_parallelism settings below) are tied to the hardware used. i.e, if we use v5p or v6e, these will be different. Don't need to update this CL, but something to remember for future CLs.

Reply:
Ack.